Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts

```diff
@@ -123,6 +123,15 @@ async function run() {
             tl.setVariable(webAppUri, publishingProfile.destinationAppUrl);
         }
 
+        if(publishingProfile && publishingProfile.destinationAppUrl) {
+            try{
+                await azureRESTUtility.warmupAzureAppService(publishingProfile.destinationAppUrl);
+                deployUtility.sleep(3000);
```

Comment:
After warming up waiting for small period of time to proceed with deployment of application. Will discuss offline over it.